samples/tutorial/sym1/clean

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#
+# clean.sh
+
+if [ -f $1.c ]; then
+   echo
+   echo "--- Cleaning $1 ---"
+
+   rm $1.s $1.o $1.map $1.bin $1.hex > /dev/null 2>&1
+
+   echo "--- Cleaned  $1 ---"
+
+fi
+
```

Comment:
We don't have/want any batch file in this repository, so please remove this file.

Reply:
**REMOVED**